cmd/sebak/cmd/wallet/pay.go

```diff
@@ -0,0 +1,103 @@
+package wallet
```

Comment:
How about rename `pay.go` to `payment.go`, we did not use the term, `pay` in this project. :)